cafebabel/articles/views.py

```diff
@@ -125,6 +125,14 @@ def draft_detail(draft_id):
     return render_template('articles/draft_detail.html', article=article)
 
 
+@draft_bp.route('/list/')
```

Comment:
Not fond of that URL, maybe we should change other views to get back `/` for that one?

Reply:
Idea: having `/new/` for the form + `POST` endpoint and keep `/` for the listing.

Reply:
Do you mean `/draft/list/` is a 404?
It would mean [we need to fix our test](https://github.com/cafebabel/cafebabel.com/pull/64/files#diff-6c262072995a4969bf7f43bbc16f4c35R330).

Reply:
No, I mean keeping the pattern `/` to display a list (here) and [using `/new/`](https://github.com/cafebabel/cafebabel.com/pull/60/files#diff-91c8d36d03e7bc8ceb0f75e804d4f939R11) to deal with creation.